libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/camera/transition/MapboxNavigationCameraTransitionTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.mapbox.navigation.ui.maps.camera.transition
+
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.plugin.animation.CameraAnimationsPlugin
+import com.mapbox.maps.plugin.animation.CameraAnimatorOptions
+import com.mapbox.navigation.ui.maps.camera.utils.normalizeBearing
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.slot
+import io.mockk.unmockkStatic
+import io.mockk.verify
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class MapboxNavigationCameraTransitionTest {
+
+    private val mapboxMap: MapboxMap = mockk()
+    private val cameraPlugin: CameraAnimationsPlugin = mockk()
+    private val transitions = MapboxNavigationCameraTransition(mapboxMap, cameraPlugin)
+
+    @Before
+    fun setup() {
+        mockkStatic("com.mapbox.navigation.ui.maps.camera.utils.MapboxNavigationCameraUtilsKt")
```

Comment:
See suggestion above

```suggestion
        mockkStatic("com.mapbox.navigation.ui.maps.camera.utils.MapboxNavigationCameraUtils")
```